src/data/faq_de.json

```
@@ -102,6 +102,148 @@
+                        ]
+                    },
+                    {
+                        "title": "Wie soll der Nachweis einer Impfung geprüft werden können?",
+                        "anchor": "vac_cert_verification",
+                        "active": true,
+                        "textblock": [
+                            "Das Impfzertifikat (QR-Code) wird z.B. über die CovPass-App oder die Corona-Warn-App (CWA) digital oder alternativ als maschinenlesbarer Ausdruck genutzt. Das Impfzertifikat enthält nur Informationen zum Impfstatus, den Namen des Geimpften und das Geburtsdatum. Für Dienstleister, die den Impfstatus überprüfen möchten, wird es eine Prüf-App zur Prüfung des Impfzertifikats geben. Damit kann der Impfstatus ähnlich wie ein Barcode eines Flug- oder Bahntickets gescannt werden. Alternativ ist ein Nachweis mit dem analogen Impfausweis möglich."
+                        ]
+                    },
+                    {
+                        "title": "Warum wurde kein EU-weites Projekt für ein Zertifikat ausgeschrieben?",
+                        "anchor": "vac_cert_eu",
+                        "active": true,
+                        "textblock": [
+                            "Eine gemeinsame EU-Ausschreibung hätte zu viel Zeit benötigt und wäre aufgrund der unterschiedlichen Impfinformationssysteme in den Mitgliedstaaten auch schwierig umzusetzen gewesen. Beim EU-Ansatz geht es um die Regelung eines Anerkennungsrahmens. Etwaige grenzüberschreitende Nutzungspiloten könnten seitens der EU-Kommission begleitet werden. Bei der Umsetzung des Digitalen Impfnachweises in Deutschland werden die EU-Vorgaben von vornherein berücksichtigt."
+                        ]
+                    },
+                    {
+                        "title": "Wirkt sich der Impfstatus auf die Berechnung des Risikos aus? Beispielsweise, wenn eine geimpfte Person sich infiziert haben sollte und dann andere Nutzer*innen warnt?",
+                        "anchor": "vac_cert_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "Nein, in der Risikoberechnung wird nicht berücksichtigt, ob Sie gegen Corona geimpft oder genesen sind. Die Funktionen sind voneinander unabhängig. Die Qualität Ihrer Risikobewertung ändert sich nicht, je nachdem, ob Sie sich dafür entscheiden, Ihren Impfausweis in der Anwendung zu speichern, was unterstreicht, dass dies völlig freiwillig ist. Ihr persönlicher Impfstatus kann jedoch Ihre Interpretation der Risiko-Warnung beeinflussen."

```

Same here, I would not mention the recovered case...
```suggestion
                            "Nein, in der Risikoberechnung wird nicht berücksichtigt, ob Sie gegen Corona geimpft sind. Die Funktionen sind voneinander unabhängig. Die Qualität Ihrer Risikobewertung ändert sich nicht, je nachdem, ob Sie sich dafür entscheiden, Ihren Impfausweis in der Anwendung zu speichern, was unterstreicht, dass dies völlig freiwillig ist. Ihr persönlicher Impfstatus kann jedoch Ihre Interpretation der Risiko-Warnung beeinflussen."
```